recipes-browser/images/core-image-wpe-crosscompilation.bb

```diff
@@ -0,0 +1,10 @@
+SUMMARY = "WPE cross-compilation image, using ARM Traditional ISA."
```

Comment:
This comment is not right, as the ISA used depends on the target machine. An image definition can't set that.

If you want to generate an SDK with thumb2 by default try setting this on the file ```conf/local.conf```
```
ARM_INSTRUCTION_SET = "thumb"
```

Reply:
Let's commit this anyway.. i can update the comment after